ddtrace/internal/telemetry/writer.py

```diff
@@ -217,17 +219,18 @@ def _restart(self):
     def disable(self):
         # type: () -> None
         """
-        Disable the telemetry collection service.
+        Disable the telemetry collection service and flush the existing integrations and events
```

Comment:
```suggestion
        Disable the telemetry collection service and drop the existing integrations and events           
```

? not sure I like "flush" since we aren't actually flushing them ?